src/Chapter4.hs

```diff
@@ -610,7 +651,9 @@ Implement the 'Monad' instance for our lists.
 🕯 HINT: You probably will need to implement a helper function (or
   maybe a few) to flatten lists of lists to a single list.
 -}
-
+instance Monad List where
+  (>>=) Empty _ = Empty
+  (>>=) xs f = listConcat $ fmap f xs
```

Comment:
Great!
You can actually remove the above Empty case, as it is handled both in `fmap` and `listConcat`, so it is safe 👍🏼